packages/web/src/components/result/ReactiveList.js

```diff
@@ -2,26 +2,27 @@ import React, { Component } from 'react';
 import hoistNonReactStatics from 'hoist-non-react-statics';
 import {
 	addComponent,
+	loadMore,
 	removeComponent,
-	setStreaming,
-	watchComponent,
+	setComponentProps,
+	setQueryListener,
 	setQueryOptions,
-	updateQuery,
-	loadMore,
+	setStreaming,
 	setValue,
-	setQueryListener,
-	setComponentProps,
 	updateComponentProps,
+	updateQuery,
+	watchComponent,
 } from '@appbaseio/reactivecore/lib/actions';
 import {
-	isEqual,
-	getQueryOptions,
-	pushToAndClause,
-	getClassName,
-	parseHits,
 	checkSomePropChange,
-	getOptionsFromQuery,
+	getClassName,
 	getCompositeAggsQuery,
+	getOptionsFromQuery,
+	getQueryOptions,
```

Comment:
@ShahAnuj2610 Why there are formatting changes?

Reply:
rearranged imports according to `alphabetical` order